package.json

```diff
@@ -30,7 +30,7 @@
   "private": true,
   "scripts": {
     "start": "react-native start",
-    "start:after-pull": "yarn install && (cd ios && pod install) && yarn generate && yarn start",
+    "start:after-pull": "yarn install && (cd ios && bundle exec pod install) && yarn generate && yarn start",
```

Comment:
What do you think about adding the command `bundle` after the `yarn install` to update the gem dependencies?
```json
"start:after-pull": "yarn install && bundle && (cd ios && bundle exec pod install) && yarn generate && yarn start",
```

Reply:
At this point, I think doing the `git pull` at the start will be helpful to ensure that we're always working with the latest version of the codebase. In case the branch is already up to date (ex. gui command), the pull will complete quickly without making any changes

Reply:
> At this point, I think doing the `git pull` at the start will be helpful to ensure that we're always working with the latest version of the codebase. In case the branch is already up to date (ex. gui command), the pull will complete quickly without making any changes

git pull fails if remote is not set up